portfolio/src/main/java/com/google/sps/user/repository/UserRepositoryFactory.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.user.repository;
+
+import com.google.sps.user.User;
+import com.google.sps.user.repository.impl.DatastoreUserRepository;
+import org.jetbrains.annotations.Nullable;
+
+public class UserRepositoryFactory{
+    public static enum UserRepositoryType{
```

Comment:
Can we have an enum RepositoryType and reuse it for both User and Comment?

Reply:
done